docs/edgeql/paths.rst

```diff
@@ -37,7 +37,7 @@ friend of another User*.
 
   select User.friends;
 
-Paths can traverse arbitrarily many links.
+Paths can arbitrarily traverse multiple links.
```

Comment:
I agree this needs a rewrite, but I think this one may change the intended meaning slightly. What about this?

```suggestion
Paths can traverse to an arbitrary depth in a series of nested links.
```

Judging based on the example, I think this gets closer to what we're trying to show.